cnames_active.js

```diff
@@ -119,6 +119,7 @@ var cnames_active = {
   "aider": "tjz101.github.io/aider-js-pages",
   "airtable-plus": "victorhahn.github.io/airtable-plus",
   "ais": "yunyoujun.github.io/ais.js",
+  "aitt": "https://aittssal.github.io/aitt/",
```

Comment:
```suggestion
  "aitt": "aittssal.github.io/aitt",
```
As the CI error notes, you don't need `https://`, nor the trailing slash